care/emr/api/viewsets/report/report_upload.py

```
@@ -0,0 +1,336 @@
+        try:
+            signed_url = instance.files_manager.read_signed_url(instance)
+
+            return Response(
+                {
+                    "download_url": signed_url,
+                    "file_name": instance.name,
+                    "mime_type": instance.meta.get("mime_type"),
+                }
+            )
+
+        except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)